models/users.js

```diff
@@ -0,0 +1,28 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/db.js';
+
+const User = sequelize.define('User', {
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    allowNull: false,
+    primaryKey: true,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
```

Comment:
can we use regex here for email pattern?

Reply:
Email pattern is enforced by "Yup" validation schema.